model_compression_toolkit/quantizers_infrastructure/trainable_infrastructure/keras/base_keras_quantizer.py

```diff
@@ -61,6 +62,23 @@ def from_config(cls, config: dict):
             # Note that a quantizer only receive quantization config and the rest of define hardcoded inside the speficie quantizer.
             return cls(quantization_config=quantization_config)
 
+        def get_trainable_variables(self, group: VariableGroup) -> List[tf.Tensor]:
+            """
+            Get trainable parameters with specific group from quantizer
+
+            Args:
+                group: Enum of variable group
+
+            Returns:
+                List of trainable variables
+            """
+            quantizer_trainable = []
+            for name, (quantizer_parameter, parameter_group) in self.quantizer_parameters.items():
```

Comment:
Why this code not in common ?

Reply:
This is because the trainable flag is different in keras and pytorch.